### PR TITLE
{#176} create fish of handler for /mywords

### PR DESCRIPTION
**Pull request by UlkaBo**

### CODE REVIEW

File: app/handlers/personal/list_of_user_words.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    """
+
+    list_of_words = await get_list_of_words(msg.from_user.id)
+    for card in list_of_words:
+        await msg.answer(
+            text=f"{card['foreign_word']}    {card['native_word']}    {card['learning_status']}"
+        )
+
+
+def register_handler_list(dp: Dispatcher) -> None:
```
Comment (YanaAnisimova) [inline app/handlers/personal/list_of_user_words.py:33]:
```suggestion
def register_handler_mywords(dp: Dispatcher) -> None:
```

File: app/handlers/personal/list_of_user_words.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from aiogram.filters import Command
+
+from aiogram import Dispatcher, types
+from aiogram.fsm.context import FSMContext
+
+from app.base_functions.list_of_words import get_list_of_words
+
+
+async def list_of_user_words(msg: types.Message, state: FSMContext) -> None:
```
[the remark this answers is not in the document]
Comment (UlkaBo) [reply app/handlers/personal/list_of_user_words.py:9]: ок

[the remark this answers is not in the document]
Comment (UlkaBo) [reply app/handlers/personal/list_of_user_words.py]: done

File: app/handlers/personal/list_of_user_words.py (hunk 1 above)
Comment (UlkaBo) [reply app/handlers/personal/list_of_user_words.py]: done

**Review by YanaAnisimova — APPROVED** [review]

**Pull request merged**